src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -437,4 +393,56 @@ private function reduceBuffer($messages)
             return substr($value, -4);
         }, array_merge(array($this->bufferedOutput->fetch()), (array) $messages));
     }
+
+    private function createBlock($messages, $type = null, $style = null, $prefix = ' ', $padding = false, $escape = false)
+    {
+        $indentLength = 0;
+        $lines = array();
+
+        if (null !== $type) {
+            $typePrefix = sprintf('[%s] ', $type);
+            $indentLength = strlen($typePrefix);
+            $lineIndentation = str_repeat(' ', $indentLength);
+        }
+
+        // wrap and add newlines for each element
+        foreach ($messages as $key => $message) {
+            if ($escape) {
+                $message = OutputFormatter::escape($message);
+            }
+
+            $lines = array_merge($lines, explode(PHP_EOL, wordwrap($message, $this->lineLength - Helper::strlen(strip_tags($prefix)) - $indentLength, PHP_EOL, true)));
+
+            // prefix each line with a number of spaces equivalent to the type length
+            if (null !== $type) {
```

Comment:
@ro0NL I tried to remove the check here and make it in the last loop e.g:

```php
foreach ($lines as &$line) {
    if ('' !== $line && null !== $type) {
        $line = $lineIndentation === substr($line, 0, $indentLength) ? $line : $lineIndentation.$line;
    }
    $line = sprintf('%s%s', $prefix, $line);
    // ...
}
```

unfortunately it breaks the line format.